src/Dashboard/WithdrawStakingRewards/WithdrawStakingRewards.tsx

```diff
@@ -225,7 +226,7 @@ export class WithdrawStakingRewards extends React.Component<Props, State> {
 
       if (isEasyConfirmationEnabled) {
         try {
-          await walletManager.ensureKeysValidity()
+          await ensureKeysValidity(walletManager._id)
```

Comment:
wallet.id

Reply:
here it is still a class, gonna be doing in your PR.